src/main/java/com/refinedmods/refinedstorage/apiimpl/autocrafting/task/v6/Craft.java

```diff
@@ -138,4 +138,8 @@ CompoundNBT writeToNbt() {
         return tag;
     }
 
+    void finishCalculation() {
```

Comment:
?

Reply:
just cleans up the call structure a bit.